SwiftRuntimeLibrary/Makefile

```diff
@@ -0,0 +1,24 @@
+TOP=..
+include $(TOP)/common.mk
+
+all: create-pinvokes
+
+clean-ios:
+	rm -rf $(TOP)/SwiftRuntimeLibrary.iOS/bin/*
+	rm -rf $(TOP)/SwiftRuntimeLibrary.iOS/GeneratedCode/*
+
+clean-mac:
+	rm -rf $(TOP)/SwiftRuntimeLibrary.Mac/bin/*
+	rm -rf $(TOP)/SwiftRuntimeLibrary.Mac/GeneratedCode/*
+
+clean: clean-ios clean-mac
+	
+create-pinvokes-ios:
+	$(shell mkdir -p $(TOP)/SwiftRuntimeLibrary.iOS/GeneratedCode)
```

Comment:
Why the use of shell??

Reply:
@mandel-macaque copying the pattern [here](https://github.com/xamarin/binding-tools-for-swift/blob/master/swiftglue/Makefile#L148). What is the alternative? (sorry I'm not super familiar with makefiles)

Reply:
When I try changing it to just `mkdir -p $(TOP)/SwiftRuntimeLibrary.iOS/GeneratedCode` (removing the `$(shell ...)`), the next command (calling Type-o-Matic) fails and says
```
Could not find a part of the path "/binding-tools-for-swift/SwiftRuntimeLibrary.iOS/GeneratedCode/BindingMetadata.iOS.cs".
```
But I can see the directory is being made; why would this happen?

Reply:
Resolved by removing `$(shell ...)` entirely; output is much nicer now!